R/package_search.R

```diff
@@ -66,11 +66,11 @@ package_search <- function(q = '*:*', fq = NULL, sort = NULL, rows = NULL,
     include_private = as_log(include_private), 
     use_default_schema = as_log(use_default_schema)
   ))
-  if (ver <= 23) {
-    args$use_default_schema <- args$include_drafts <- args$include_private <- NULL
+  if (ver < 23.5) {
+    args$include_drafts <- args$use_default_schema <- args$include_private <- NULL
+  } else if (ver < 26.1) {
```

Comment:
what if this 2nd if statement is not TRUE?

Reply:
i guess there would be no change to those three parameters, they'd be whatever the user supplied or the defaults. i can spell it out with an explicit `else args <- args`

Reply:
oh right, its okay if args is. not modified. makes sense now